docs/dev/cli/sauce-connect-proxy.md

```diff
@@ -57,6 +59,7 @@ __Shorthand__: `-c`
 
 __Description__: Sets your Sauce Labs region data center such as `us-west`, `eu-central`, etc... For more information, see [Data Center Endpoints](#data-center-endpoints). <br/>
 __Default__: If you don't specify a Data Center at all, Sauce Connect will default to `us-west`. <br/>
+__Enviroment variable__: `SAUCE_REGION`<br/>
```

Comment:
Hi @waggledans , I've seen this referenced in code snippets as `SAUCE_DC`. Does that apply as well?

cc @sweeneyskirt-sl